packages/suite/src/views/wallet/coinmarket/savings/setup/components/Summary/index.tsx

```diff
@@ -58,7 +59,10 @@ const Summary = ({
         </Left>
         <Right>
             <Fiat>
-                <FormattedNumber currency={fiatCurrency} value={annualSavingsFiatAmount} />
+                <FormattedFiatAmount
+                    currency={fiatCurrency as string}
```

Comment:
Yes, I notices that it's marked as optional many levels up and that it shouldn't be the case but decided not to refactor. I suppose a check hasn't been done in some parent component and instead of the check someone just marked it as optional everywhere. I have seen it before this PR, things that should be 100% defined in some child components still being marked as optional because of the initial `type | undefined` type :( I'd keep it for now to not introduce more refactors

Reply:
Made optional, added a check in the `FormattedFiatAmount`